app/controllers/api/person_escort_records_controller.rb

```diff
@@ -17,9 +22,14 @@ def create
       render_person_escort_record(person_escort_record, :created)
     end
 
-    def show
-      person_escort_record = PersonEscortRecord.find(params[:id])
+    def update
+      PersonEscortRecords::ParamsValidator.new(update_person_escort_record_attributes).validate!
+      person_escort_record.confirm!(update_person_escort_record_attributes)
```

Comment:
is an update _always_ a confirmation ?

Reply:
yes, a client can only confirm a PER, and can't change anything else on the status/attributes on the record